translations/de/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,169 @@
+# German translations for PROJECT.
+# Copyright (C) 2022 CERN
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PROJECT VERSION\n"
+"Report-Msgid-Bugs-To: info@zenodo.org\n"
+"POT-Creation-Date: 2022-11-11 14:07+0100\n"
+"PO-Revision-Date: 2022-11-11 14:10+0100\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language: de\n"
+"Language-Team: de <LL@li.org>\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Babel 2.11.0\n"
+
+#: /Users/ppanero/Workspace/zenodo/zenodo-rdm/invenio.cfg:88
```

Comment:
minor: Also has some original path-specific things which probably shouldn't be here